src/content/blog/2025/12/03/critical-security-vulnerability-in-react-server-components.md

```diff
@@ -94,11 +94,9 @@ npm install @vitejs/plugin-rsc@latest
 
 ### Expo {/*expo*/}
 
-Upgrade to the latest `react-server-dom-webpack`:
+You only need to take action if you're using experimental RSC or Server Functions support in production. API routes are not affected. The impact on EAS Hosting was already mitigated through a WAF rule.
```

Comment:
> The impact on EAS Hosting was already mitigated through a WAF rule.

I'd recommend softening that language. It's just one layer and people should upgrade immediately regardless of any WAF rules.